### PR TITLE
Bug fix: allow local e2e testing

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Bug fix: allow local e2e testing

## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: tests/e2e/test_api.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 # OLS_URL env only needs to be set when running against a local ols instance,
 # when ols is run against a cluster the url is retrieved from the cluster.
 ols_url = os.getenv("OLS_URL")
-if "localhost" not in ols_url:
+if ols_url is not None and "localhost" not in ols_url:
```
[the remark this answers is not in the document]
Comment (bparees) [reply tests/e2e/test_api.py:31]: and under what condition is OLS_URL None?

[the remark this answers is not in the document]
Comment (bparees) [reply tests/e2e/test_api.py:31]: yes, but the http client object still has to get setup, which still requires a url, so if the url is None, what happens w/ the http client?

[the remark this answers is not in the document]
Comment (tisnik) [reply tests/e2e/test_api.py:31]: well locally I used just to run `make test-e2e` w/o the need to use any helper script (that not set up the value).

I guess we just need to warn if the value is not set, and that's it?

[the remark this answers is not in the document]
Comment (bparees) [reply tests/e2e/test_api.py:31]: or you can default it to localhost if you'd like, by modifying this line:
```
ols_url = os.getenv("OLS_URL")
```

all our actual e2e jobs that run e2e tests should be setting OLS_URL explicitly to the correct value.

[the remark this answers is not in the document]
Comment (tisnik) [reply tests/e2e/test_api.py]: done

**Pull request merged**